modules/ROOT/pages/proc_defining-and-creating-pipelineresources.adoc

```diff
@@ -76,7 +76,7 @@ spec:
 $ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/pipeline/resources.yaml
```

Comment:
Should we switch to using `tkn resource create` instead for creating resources?

Reply:
Indepedently from your comment, I created https://github.com/openshift/pipelines-tutorial/issues/60 :)